tests/test_models.py

```diff
@@ -44,6 +44,18 @@ def test_daily_min(test, expected):
     from inflammation.models import daily_min
     npt.assert_array_equal(daily_min(test), expected)
 
+@pytest.mark.parametrize(
+    "test, expected",
+    [
+        ([[0, 0], [0, 0], [0, 0]], [0, 0]),
+        ([[1, 2], [3, 4], [5, 6]], [1, 2]),
+    ]
+)
+def test_daily_stddev(test, expected):
+    """Test that min function works for an array of zeros and integers."""
```

Comment:
Docstring mentions min function, not std
```suggestion
    """Test that std function works for an array of zeros and integers."""
```